### DISCUSSION TITLE
Idea to handle overflow due to several values

### DISCUSSION THREAD

**Discussion by wenstarner** [khoait/DCE.PCF#129] [Ideas] [answered]
Hi, when testing out this component, I've noticed that multiple options will increase the size of the 'selection box', like this:
![image](https://github.com/user-attachments/assets/1142e26d-e982-4d5b-9d66-0e20d31864b3)

I know this is an extreme example, but I was wondering if you might consider a 'graceful' way to handle overflow? For example, modelled on how the first-party Field Service component works:

1. Shows a '+X' button to handle any values that don't sit on a single line:
![image](https://github.com/user-attachments/assets/ba219ba0-cad0-4a8f-b15a-c479d3e76e01)

2. Component expands when the '+X' link is clicked:
![image](https://github.com/user-attachments/assets/70e01bea-1ca4-46e3-ae22-5187e1982dec)

And then perhaps also a tooltip with the other values, when the '+X' link is hovered over.

Also, showing entity icons in front of the value (as an option?) could be a nice touch. Just thought I'd mention, in case you also like the idea.

Just thought I'd ask... given the quality of your work, if there is anyone that can implement an amazing alternative to MS' strange lack of this capability, it is you!

Thanks so much. Cheers!

**Comment by khoait** [comment, marked as answer]
Thanks for your interest and ideas!

it looks nice and clean but that's an extra click for user to view or edit the selection I think. It could be an additional prop to enable it. I'll add it to the backlog.

regarding entity icons, I'm planning to implement it similar to the Lookdown component which also supports entity image.

I'll let you know when it is available.

Cheers!

**Reply by wenstarner** [reply]
Thank so much for that! I appreciate you taking the time to respond, and for considering my feedback 🙏
